modules/varnish/templates/default.vcl

```diff
@@ -447,7 +447,7 @@ sub vcl_deliver {
 		set resp.http.Age = 0;
 	}
 
-	if (req.url ~ "^(/w/api\.php*|/w/index\.php\?title\=Special\:|/wiki/Special\:|/w/index\.php\?title\=Special%3A|/wiki/Special%3A).+$") {
+	if (req.url ~ "^(/w/api\.php*|/w/index\.php\?*|/wiki/Special\:|/wiki/Special%3A).+$") {

```

Comment:
Not `*` just `/w/index\.php\?` would be enough I think. Either that or maybe `/w/index\.php*`, which might be better?

Reply:
Do api/rest need indexing?

Reply:
Api is already there.

Reply:
Cant api and index be merged into one (api|index|rest)\.php* statement?

Reply:
Use `^(/w/(api|index|rest)\.php*|/wiki/Special(\:|%3A)).+$` I believe